docs/source/json-api/index.rst

```diff
@@ -1234,6 +1234,401 @@ HTTP Response
       "status": 200
     }
 
+
+Creating a New User
+********************
+
+This endpoint exposes the Ledger API's :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>`.
+
+HTTP Request
+============
+
+- URL: ``/v1/user/create``
+- Method: ``POST``
+- Content-Type: ``application/json``
+- Content:
+
+.. code-block:: json
+
+    {
+      "userId": "Carol",
+      "primaryParty": "Carol",
+      "rights": [
+        {
+          "type": "CanActAs",
+          "party": "Carol"
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Alice",
+        },
+        {
+          "type": "CanReadAs",
+          "party": "Bob",
+        },
+        {
+          "type": "ParticipantAdmin"
+        }
+      ]
+    }
+
+Please refer to :ref:`CreateUserRequest <com.daml.ledger.api.v1.admin.user_management_service.CreateUserRequest>` documentation for information about the meaning of the fields.
+
+All fields in the request are required, this means that an empty JSON object is not a valid request to allocate a new user.
+
+HTTP Response
+=============
+
+.. code-block:: json
+
+    {
+      "result": true,
```

Comment:
This is a bit confusing. What does `result: false` mean? Wouldn't the caller get an HTTP error?

Suggestoin: use the same result type as the gRPC API: the created user. This way future changes of the creation operations that add extra information to the user on the server-side will be nicely covered.

Reply:
We don't automatically include results from the gRPC API, as explained [#7 here](https://github.com/digital-asset/daml/blob/9c03e7908bbd85ab1f8686cc581d49cc93beea92/ledger-service/http-json/README.md#development-guidelines).

As it happens `true` here is used not as part of the boolean type but the singleton type `true`; `false` is not in the codomain. So another representation of `unit` might be good to return, e.g. `{}`, which nicely affords future extension where we may not return anything now.

Reply:
I've adjusted that in a not yet merged [PR](https://github.com/digital-asset/daml/pull/12578).